inst/rmd/ioslides/default.html

```diff
@@ -92,6 +92,22 @@
 /* https://github.com/ropensci/plotly/pull/524#issuecomment-468142578 */
 slide:not(.current) .plotly.html-widget{
   display: block;
+
+  code{white-space: pre-wrap;}
+  span.smallcaps{font-variant: small-caps;}
+  span.underline{text-decoration: underline;}
+  div.column{display: inline-block; vertical-align: top; width: 50%;}
+  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
+  ul.task-list{list-style: none;}
+  $if(quotes)$
+  q { quotes: "“" "”" "‘" "’"; }
+  $endif$
+  $if(highlighting-css)$
+  $highlighting-css$
+  $endif$
+  $if(displaymath-css)$
+  .display.math{display: block; text-align: center; margin: 0.5rem auto;}
+  $endif$
```

Comment:
Are you sure this has to be added inside this selector ? 
I believe those styles is for the whole document instead, right ? 

Just a bracket issue I think